File: Java/src/Resultado.java
public class Resultado {
    private long idTarefa;
    private int resultado;
    private long tempo;

    public Resultado(long idTarefa, int resultado, long tempo) {
        this.idTarefa = idTarefa;
        this.resultado = resultado;
        this.tempo = tempo;
    }

    public long getIdTarefa() {
        return idTarefa;
    }

    public int getResultado() {
        return resultado;
    }

    public long getTempo() {
        return tempo;
    }
}

File: GO/src/classes/Resultado.go
package classes

type Resultado struct{
	IdTarefa int64
	Resultado int
	Tempo int64
}

func CriarResultado(idTarefa int64, resultado int, tempo int64) *Resultado{
	return &Resultado{
		IdTarefa: idTarefa,
		Resultado: resultado,
		Tempo: tempo,
	}
}
